src/page/blog/Naming.js

```diff
@@ -91,7 +91,7 @@ class Naming extends Component {
                         </p>
 
                         <InlineEditor>
-                            {`def mapInPlace(f: a -> a, a: Array[a]): Unit & Impure`}
+                            {`def mapInPlace(f: a -> a \\ ef, a: Array[a]): Unit \\ IO`}
```

Comment:
Would `mapInPlace` be impure, if `f` is pure?

Reply:
Given that `mapInPlace` returns `Unit`

Reply:
Yes, it should be IO as you have it.